pygmt/base_plotting.py

```diff
@@ -275,6 +276,13 @@ def grdcontour(self, grid, **kwargs):
         {G}
         {U}
         {W}
+        label : str
+            Add a legend entry for the contour being plotted. Normally, the
+            annotated contour is selected for the legend. You can select the
+            regular contour instead, or both of them, by considering the label
+            to be of the format [*annotcontlabel*][/*contlabel*]. If either
+            label contains a slash (/) character then use ``|`` as the
```

Comment:
```suggestion
            label contains a slash (/) character then use | as the
```
Do we need the backticks here?

Reply:
Unnecessary. To me, backticks make the vertical bar more visible.

Reply:
Ok, let's keep it then.